package.json

```diff
@@ -26,30 +26,30 @@
     "joi": "^17.2.1"
```

Comment:
Are there any vulnerabilities being flagged on this install? I see snyk has opened some PRs on this repo for `aws-sdk`

Reply:
I didnt have any when I made the changes, I will rerun the audit

Reply:
Strangely running snyk on the repo doesnt reveal any vulns, I'm going to have a little dig into this and see what might be happening